README.md

```diff
@@ -0,0 +1 @@
+A trail task from Microverse in cooperation with wsyswsys css by wsyswsys https://github.com/wsyswsys html by moathal https://github.com/Moathal
```

Comment:
- Please use the standard README.md file that was provided by Microverse and fill it according to your project
- Add some details according to the fields available there.